docs/coding-conventions.md

```diff
@@ -320,6 +320,98 @@ coding conventions](#detailed-coding-conventions).
       subdirectories.
 - Document directories and filenames should use dashes rather than underscores.
 
+# Writing Tests
+## Writing functional tests
+### Test Interrupts
+To make sure that any asynchronous operation is canceled when the test is
+terminated, make sure to use golang contexts properly. See more details in 
+[ginkgo documentation](https://onsi.github.io/ginkgo/#spec-timeouts-and-interruptible-nodes)
+
+Avoid using new contexts (e.g. `context.Background()` or `context.TODO()`).
+Instead, prefer using the context injected by ginkgo.
+
+First, add the `ctx` optional parameter of the anonymous functions in `It`, `BeforeEach`,
+`AfterEach` or `DescribeTable`. Then use this parameter for all the contexts in this container.
```

Comment:
`Then use this parameter for all places where a context is needed in this container.`